recipe/meta.yaml

```diff
@@ -154,16 +153,48 @@ test:
     - pyomo.solvers.plugins.solvers
     - pyomo.util
     - pyomo.version
-
+  source_files:
+  # Tests are spread across many files throughout the source code.
+    - .
+  requires:
+    - pip
+    - pytest
+    - parameterized
+    - pybind11
   commands:
     - pyomo --help
+    - pip check
+    {% set tests_to_skip = "abstract_ch_pyomo_buildactions_fails_sh" %}
+    {% set tests_to_skip = tests_to_skip + " or abstract_ch_pyomo_model" %}
+    # Missing file "lsb_release"
```

Comment:
§if we add `distro` as a dependency, it should work, see https://github.com/Pyomo/pyomo/blob/1ada528693d8879f17978fe47263f8bb2b467b0a/pyomo/common/download.py#L154-L155.